website/src/lapis/getCladeLineages.ts

```diff
@@ -0,0 +1,72 @@
+import axios from 'axios';
+import { z } from 'zod';
+
+import { getClientLogger } from '../clientLogger.ts';
+
+const logger = getClientLogger('getCladeLineages');
+
+const aggregatedSchema = z.object({
+    data: z.array(
+        z
+            .object({
+                count: z.number(),
+            })
+            .catchall(z.union([z.string(), z.number(), z.boolean(), z.null()])),
+    ),
+});
+
+/**
+ * Finds the lineage definition belonging to clades, by looking for the lineage that is most
+ * commonly associated with each clade.
+ * Returns a map, mapping all clades found in the data to their respective lineage.
+ */
+export async function getCladeLineages(
+    baseUrl: string,
+    cladeField: string,
+    lineageField: string,
+): Promise<Record<string, string>> {
+    const url = `${baseUrl.replace(/\/$/, '')}/sample/aggregated`;
+    const body = {
+        fields: [cladeField, lineageField],
+        orderBy: [cladeField],
```

Comment:
```suggestion
        orderBy: [cladeField, { field: 'count', type: 'descending' }],
```

In POST requests, LAPIS can do the sorting already.

<img width="639" height="313" alt="Image" src="https://github.com/user-attachments/assets/e8e8425c-0b82-4087-b6d6-5d2dea5977a7" />

Reply:
https://github.com/GenSpectrum/dashboards/pull/842/commits/08c5c48aed4681850e191c26a1e3d0e906829a90